src/main/java/com/notably/view/MainWindow.java

```diff
@@ -32,16 +32,24 @@
 
     // Independent View parts residing in this View container
     private HelpWindow helpWindow;
+    private SideBarTreeViewImpl sidebarTreeView;
+    private BlockContentViewImpl blockcontentView;
 
     @FXML
     private StackPane commandBoxPlaceholder;
 
     @FXML
     private MenuItem helpMenuItem;
 
+    @FXML
+    private StackPane sideBarPlaceholder;
+
     @FXML
     private StackPane statusbarPlaceholder;
 
+    @FXML
+    private StackPane textFieldPlaceholder;
```

Comment:
Should we call this `blockContentPlaceholder`? In the future, we can stack either `BlockContentEditView` or `BlockContentDisplayView` on top of this placeholder (pretty sure you thought about this, just reiterating LOL!)

Reply:
Calling it blockContentPlaceholder. Thanks, I understand why this will be useful in the future :)